lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      TEST_METHOD_PREFIX = 'test_'.freeze
+      TEST_FILE_PATTERN  = './test/**/{test_*,*_test}.rb'.freeze
```

Comment:
Here are the gems I checked btw:

1. [rake](https://rubygems.org/gems/rake)
1. [rack](https://rubygems.org/gems/rack)
1. [activesupport](https://rubygems.org/gems/activesupport)
1. [mime-types](https://rubygems.org/gems/mime-types)
1. [nokogiri](https://rubygems.org/gems/nokogiri)
1. [arel](https://rubygems.org/gems/arel)
1. [sass](https://rubygems.org/gems/sass)
1. [sprockets](https://rubygems.org/gems/sprockets)
1. [minitest](https://rubygems.org/gems/minitest)
1. [rspec-expectations](https://rubygems.org/gems/rspec-expectations)
1. [rspec-core](https://rubygems.org/gems/rspec-core)
1. [rspec-mocks](https://rubygems.org/gems/rspec-mocks)
1. [rdoc](https://rubygems.org/gems/rdoc)
1. [thread_safe](https://rubygems.org/gems/thread_safe)
1. [slop](https://rubygems.org/gems/slop)
1. [rubyzip](https://rubygems.org/gems/rubyzip)
1. [haml](https://rubygems.org/gems/haml)
1. [journey](https://rubygems.org/gems/journey)
1. [sqlite3](https://rubygems.org/gems/sqlite3)
1. [newrelic_rpm](https://rubygems.org/gems/newrelic_rpm)
1. [rubygems-update](https://rubygems.org/gems/rubygems-update)
1. [fog](https://rubygems.org/gems/fog)
1. [netrc](https://rubygems.org/gems/netrc)
1. [launchy](https://rubygems.org/gems/launchy)
1. [sdoc](https://rubygems.org/gems/sdoc)
1. [bootstrap-sass](https://rubygems.org/gems/bootstrap-sass)
1. [webmock](https://rubygems.org/gems/webmock)
1. [loofah](https://rubygems.org/gems/loofah)
1. [chronic](https://rubygems.org/gems/chronic)
1. [net-http-persistent](https://rubygems.org/gems/net-http-persistent)
1. [connection_pool](https://rubygems.org/gems/connection_pool)
1. [rails-dom-testing](https://rubygems.org/gems/rails-dom-testing)
1. [rails-html-sanitizer](https://rubygems.org/gems/rails-html-sanitizer)
1. [dalli](https://rubygems.org/gems/dalli)
1. [fog-core](https://rubygems.org/gems/fog-core)
1. [activerecord-deprecated_finders](https://rubygems.org/gems/activerecord-deprecated_finders)
1. [fog-brightbox](https://rubygems.org/gems/fog-brightbox)
1. [fog-json](https://rubygems.org/gems/fog-json)
1. [sexp_processor](https://rubygems.org/gems/sexp_processor)
1. [hitimes](https://rubygems.org/gems/hitimes)
1. [airbrake](https://rubygems.org/gems/airbrake)
1. [net-ssh-multi](https://rubygems.org/gems/net-ssh-multi)
1. [byebug](https://rubygems.org/gems/byebug)
1. [colorize](https://rubygems.org/gems/colorize)
1. [ruby_parser](https://rubygems.org/gems/ruby_parser)
1. [sidekiq](https://rubygems.org/gems/sidekiq)
1. [trollop](https://rubygems.org/gems/trollop)
1. [slim](https://rubygems.org/gems/slim)
1. [sequel](https://rubygems.org/gems/sequel)
1. [parser](https://rubygems.org/gems/parser)
1. [rvm](https://rubygems.org/gems/rvm)
1. [pry-rails](https://rubygems.org/gems/pry-rails)
1. [oj](https://rubygems.org/gems/oj)
1. [ci_reporter](https://rubygems.org/gems/ci_reporter)
1. [fog-softlayer](https://rubygems.org/gems/fog-softlayer)
1. [debugger](https://rubygems.org/gems/debugger)
1. [mechanize](https://rubygems.org/gems/mechanize)
1. [kramdown](https://rubygems.org/gems/kramdown)
1. [posix-spawn](https://rubygems.org/gems/posix-spawn)
1. [fog-xml](https://rubygems.org/gems/fog-xml)
1. [simple_oauth](https://rubygems.org/gems/simple_oauth)
1. [fog-sakuracloud](https://rubygems.org/gems/fog-sakuracloud)
1. [request_store](https://rubygems.org/gems/request_store)
1. [hoe](https://rubygems.org/gems/hoe)
1. [hiredis](https://rubygems.org/gems/hiredis)
1. [ruby-prof](https://rubygems.org/gems/ruby-prof)
1. [fog-profitbricks](https://rubygems.org/gems/fog-profitbricks)
1. [liquid](https://rubygems.org/gems/liquid)
1. [net-http-digest_auth](https://rubygems.org/gems/net-http-digest_auth)
1. [fog-terremark](https://rubygems.org/gems/fog-terremark)
1. [rails_12factor](https://rubygems.org/gems/rails_12factor)
1. [fog-voxel](https://rubygems.org/gems/fog-voxel)
1. [ruby2ruby](https://rubygems.org/gems/ruby2ruby)
1. [fog-vmfusion](https://rubygems.org/gems/fog-vmfusion)
1. [omniauth-facebook](https://rubygems.org/gems/omniauth-facebook)
1. [ruby-openid](https://rubygems.org/gems/ruby-openid)
1. [mustache](https://rubygems.org/gems/mustache)
1. [fog-ecloud](https://rubygems.org/gems/fog-ecloud)
1. [fog-storm_on_demand](https://rubygems.org/gems/fog-storm_on_demand)
1. [friendly_id](https://rubygems.org/gems/friendly_id)
1. [fog-atmos](https://rubygems.org/gems/fog-atmos)
1. [mono_logger](https://rubygems.org/gems/mono_logger)
1. [active_model_serializers](https://rubygems.org/gems/active_model_serializers)
1. [sanitize](https://rubygems.org/gems/sanitize)
1. [fog-serverlove](https://rubygems.org/gems/fog-serverlove)
1. [gem-wrappers](https://rubygems.org/gems/gem-wrappers)
1. [less-rails](https://rubygems.org/gems/less-rails)
1. [sshkit](https://rubygems.org/gems/sshkit)
1. [retriable](https://rubygems.org/gems/retriable)
1. [Ascii85](https://rubygems.org/gems/Ascii85)
1. [redis-store](https://rubygems.org/gems/redis-store)
1. [fog-riakcs](https://rubygems.org/gems/fog-riakcs)
1. [ZenTest](https://rubygems.org/gems/ZenTest)
1. [rack-contrib](https://rubygems.org/gems/rack-contrib)
1. [rack-cors](https://rubygems.org/gems/rack-cors)
1. [aasm](https://rubygems.org/gems/aasm)
1. [pry-byebug](https://rubygems.org/gems/pry-byebug)
1. [redis-activesupport](https://rubygems.org/gems/redis-activesupport)
1. [redis-rack](https://rubygems.org/gems/redis-rack)
1. [redis-actionpack](https://rubygems.org/gems/redis-actionpack)